index.html

```diff
@@ -10,47 +10,47 @@
-      </header>
+      <ul class="nav-menu">
+        <li class="nav"><a href="#myportfolio">Portfolio</a></li>
+        <li class="nav"><a href="#myabout">About</a></li>
+        <li class="nav"><a href="#mycontact">Contact</a></li>
+      </ul>
+      <div class="hamburgers">
+        <span class="bar"></span>
+        <span class="bar"></span>
+        <span class="bar"></span>
+      </div>
+    </nav>
+  </header>
 
-    <main class="coverAll-con">
-      <section class="img-bg" id="myportfolio">
-        <div class="container">
-          <div class="bio">
-            <h1>I'm Amaka Glad to see you! </h1>
-          </div>
-          <div class="paragraph">
-            <p>I'm a software developer!  I can help you build a product, feature your website Look through some
-              of my work and experience! If you like what you see and have a project you need coded, don't hesitate to
-              contact me.</p>
-          </div>
-          <div class="socials">
-            <h2 class="lets-connect">LET'S CONNECT</h2>
-            <ul class="social-links">
-              <li><a href="https://twitter.com/lawrah_xo"><img src="images/twitter.jpg" alt="twitter"></a></li>
-              <li><a href="https://www.linkedin.com/in/laura-konwea-99647a23a/"><img src="images/Linkedin icon.jpg"
-                    alt="linkedin"></a></li>
-              <li><a href="#"><img src="images/medium.jpg" alt="medium"></a></li>
-              <li><a href="https://github.com/lawrahkonwea"><img src="images/github.jpg" alt="github"></a></li>
-              <li><a href="#"><img src="images/angellist.jpg" alt="angellist"></a></li>
-            </ul>
-          </div>
+  <main class="coverAll-con">
+    <section class="img-bg" id="myportfolio">
+      <div class="container">
+        <div class="bio">
+          <h1>I'm Amaka Glad to see you! </h1>
+        </div>
+        <div class="paragraph">
+          <p>I'm a software developer! I can help you build a product, feature your website Look through some
+            of my work and experience! If you like what you see and have a project you need coded, don't hesitate to
+            contact me.</p>
         </div>
-      </section> 
+        <div class="socials">
+          <h2 class="lets-connect">LET'S CONNECT</h2>
+          <ul class="social-links">
+            <li><a href="https://twitter.com/lawrah_xo"><img src="images/twitter.jpg" alt="twitter"></a></li>
+            <li><a href="https://www.linkedin.com/in/laura-konwea-99647a23a/"><img src="images/Linkedin icon.jpg"
+                  alt="linkedin"></a></li>
+            <li><a href="#"><img src="images/medium.jpg" alt="medium"></a></li>
+            <li><a href="https://github.com/lawrahkonwea"><img src="images/github.jpg" alt="github"></a></li>
+            <li><a href="#"><img src="images/angellist.jpg" alt="angellist"></a></li>
+          </ul>
+        </div>
+      </div>
+    </section>
     <section class="project-main-wrapper1">
```

Comment:
- [ ] Kindly display the whole of this project section dynamically with javascript. By doing this, you will only need to add the data in the array in the future when you want to add a new project. So to do this, you will have to create an array, just as you did in the pop-up. Then iterate on it with a loop. Kindly remember all the files in the HTML file too.

Reply:
Fixed.